dags/analyzer_helper/tests/integration/test_discord_transform_raw_data.py

```diff
@@ -0,0 +1,158 @@
+import unittest
+from datetime import datetime
+
+from dags.analyzer_helper.discord.discord_transform_raw_data import DiscordTransformRawData
+
+
+class TestDiscordTransformRawData(unittest.TestCase):
+
+    def setUp(self):
+        self.transformer = DiscordTransformRawData()
+        self.platform_id = 'discord_platform'
+        self.period = datetime(2023, 1, 1)
+
+    def test_transform_data_with_replied(self):
+        raw_data = [
```

Comment:
All fields of raw data should be available. link to it [ref](https://rndadocs.notion.site/Analyzer-Discord-Pipeline-5009512d82a243deb35b3388e17b29c5#:~:text=An%20example%20for%20rawinfos%20document)